velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/exec/fuzzer/WindowFuzzer.h"
+
+#include "velox/exec/tests/utils/PlanBuilder.h"
+
+DEFINE_bool(
+    enable_window_reference_verification,
```

Comment:
Do we need this flag? We know that DuckDB results for window functions do not match Presto, hence, we cannot use DuckDB anyway.

Reply:
Hi @mbasmanova, I'd prefer to keep this flag in the current PR for two reasons.
1. This flag is needed when comparing the result with Presto that will be added very soon.
2. Removing this flag requires large changes to the verifyWindow() method and the update logic for statistics. This would actually make it harder to review this PR because the current code for verifyWindow() was moved from AggregationFuzzer without much change and should be simple to review.